src/main/java/frc/robot/subsystems/shooter/commands/Shoot.java

```diff
@@ -36,7 +36,7 @@ public Shoot(Shooter shooter, DoubleSupplier distance) {
      * @return 15. [rpm]
      */
     public static double getSetpointVelocity(double distance) {
-        return 100;
+        return 6000 * Shooter.velocity.getDouble(0);
```

Comment:
Magic number.

Reply:
It's just for testing purposes, there is no real need to do something with it in my opinion (as it will be deleted when we will calibrate the shooter)